src/components/config/validate.ts

```diff
@@ -288,7 +290,13 @@ export const validateConfig = (configuration: Config): Validation => {
 
       if (!url) return PROBE_REQUEST_NO_URL
 
-      if (url && !isValidURL(url)) return PROBE_REQUEST_INVALID_URL
+      if (url) {
+        if (request.ping === true) {
```

Comment:
how about having and instead of empty if like this?
for example: `if (request.ping !== true && !isValidURL(url))`